vassal-app/src/main/java/VASSAL/build/module/map/MenuDisplayer.java

```diff
@@ -49,7 +49,7 @@
 import VASSAL.counters.Properties;
 
 public class MenuDisplayer extends MouseAdapter implements Buildable {
-  public static final Font POPUP_MENU_FONT = new Font("Dialog", Font.PLAIN, 10);
+  public static final Font POPUP_MENU_FONT = new Font("Dialog", Font.PLAIN, 10);  //NON-NLS
 
```

Comment:
I actually change any of these
new Font("Dialog", Font.PLAIN, 10)
I find to
new Font(Font.DIALOG, Font.PLAIN, 10)
Saves on a NON-NLS comment :)
Not vital at all.